codebundles/k8s-postgres-query/runbook.robot

```diff
@@ -1,19 +1,46 @@
 *** Settings ***
-Metadata          Author    Jonathan Funk
-Documentation     Runs a postgres SQL query and pushes the returned query result as an SLI metric.
-...               During execution, the SQL query should be passed to a Kubernetes workload that has access to the psql binary.
-...               The workload will run the query and return the result from stdout.
-Force Tags        K8s    Kubernetes    Postgres    SQL    database    psql
-Suite Setup       Suite Initialization
-Library           RW.Core
-Library           RW.K8s
-Library           RW.Postgres
-Library           RW.Utils
-Library           RW.platform
+Documentation       Runs a postgres SQL query and pushes the returned result into a report.
+...                 During execution, the SQL query should be passed to a Kubernetes workload that has access to the psql binary.
+...                 The workload will run the query and return the results from stdout.
+Metadata            Author    Shea Stewart
+
+Library             RW.Core
+Library             RW.K8s
+Library             RW.Postgres
+Library             RW.Utils
+Library             RW.platform
+
+Suite Setup         Suite Initialization
+
+Force Tags          k8s    kubernetes    postgres    sql    database    psql
+
+
+*** Tasks ***
+Run Postgres Query And Results to Report
+    ${templated_query}=    RW.Postgres.Template Command
+    ...    query=${QUERY}
+    ...    hostname=${HOSTNAME}
+    ...    database=${psql_database}
+    ...    username=${psql_username}
+    ...    password=${psql_password}
+    ...    report=true
```

Comment:
Nit picking: curious how you feel about the readability of type casting modes like `report` to bool. According to https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#argument-conversion we'd need to pass the value as TRUE / ON / YES / 1 in order for it to accurately parse to a bool. I'm very python-oriented so I like to keep my function signatures & type hints accurate but if you find dealing with strings more intuitive I can ease up on that.

Reply:
This was one of those cases where it started out as bool, didn't work immediately, tested as string, worked, moved on with life.

Reply:
For what it's worth, I'd like to refine this down the road to allow the user to input their own optional command arguments, but the escaping was a real pain with psql and so for now it was easier to set the report option and then manually add \t \a psql commands.

Reply:
This is updated to bool now, tested successfully